lib/main.dart

```diff
@@ -15,23 +15,25 @@ import 'package:custodes/vista/splashscreen.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 import 'package:firebase_core/firebase_core.dart';
 import 'package:custodes/modelo/firebase_options.dart';
-import 'package:flutter_dotenv/flutter_dotenv.dart' show dotenv;
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
   // Inicializa Firebase
-  await Firebase.initializeApp(
-    options: DefaultFirebaseOptions.currentPlatform,
-  );
+  try {
+    await Firebase.initializeApp(
+      options: DefaultFirebaseOptions.currentPlatform,
+    );
+  } catch (e) {
+    throw ("Error al inicializar Firebase: $e");
+  }
 
   // Emulador de Firebase: true para usar el emulador, false para usar la base de datos en la nube
   // No usar la nube porque +10 mensajes por mes cuestan dinero :'D
-  const bool useEmulator = true;
+  const bool useEmulator = false;
```

Comment:
Good for now while we test android, change afterwards!!

Reply:
bet